inc/computerantivirus.class.php

```diff
@@ -253,18 +253,18 @@ function showForm($ID, $options = []) {
       echo "<tr class='tab_bg_1'>";
       echo "<td>".Computer::getTypeName(1)."</td>";
       echo "<td>".$comp->getLink()."</td>";
-      if (Plugin::haveImport()) {
-         echo "<td>".__('Automatic inventory')."</td>";
-         echo "<td>";
-         if ($ID && $this->fields['is_dynamic']) {
+      echo "<td>".__('Automatic inventory')."</td>";
+      echo "<td>";
+      if ($ID && $this->fields['is_dynamic']) {
+         if (Plugin::haveImport()) {
```

Comment:
`autoinventory_information` may output nothing, as hook will, in most case, mapped to only a few item types.
Maybe something like this could be better on functionnal point of view:
```php
ob_start();
Plugin::doHook("autoinventory_information", $this);
$info = ob_get_clean();
if (empty($info)) {
   $info = __('Yes');
}
echo $info;
```

I removed the `Plugin::haveImport()` condition as I noticed that some plugins has a `autoinventory_information` hook without having a `import_item` hook (xivo plugin is in this case).

I guess this could be mutualized for all assets.

Reply:
I've reworked this part, in two separate commits to make things clear